packages/app-accounts/src/modals/Forgetting.tsx

```diff
@@ -61,7 +60,13 @@ class Forgetting extends React.PureComponent<Props> {
           {t('Confirm account removal')}
         </Modal.Header>
         <Modal.Content>
-          <AddressSummary value={address} />
+          <AddressRow
+            isInline
+            value={address}
+          >
+            <p>{t('You are about to remove this account from your list of available accounts. Once completed, should you need access again, you need to re-create the account either via seed or via a backup file.')}</p>
```

Comment:
Suggestions to make it easier to read
```suggestion
            <p>{t('You are about to remove this account from your list of available accounts. Once completed, should you need to access it again, you will have to re-create the account either via seed or via a backup file.')}</p>
```